js/dom_types.ts

```diff
@@ -90,7 +90,7 @@ export interface ProgressEventInit extends EventInit {
   total?: number;
 }
 
-export interface URLSearchParams {
+export interface URLSearchParams extends DomIterable<string, string>{
```

Comment:
I don't think this has been through the formatter again.  Prettier would have inserted a space before the opening brace.